### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by mellogarrett**
Better readability

### CODE REVIEW

File: src/content/concepts/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ At its core, *webpack* is a _static module bundler_ for modern JavaScript applic
 
 T> Learn more about JavaScript modules and webpack modules [here](/concepts/modules).
 
-Since v4.0.0 webpack does not require a configuration file. Nevertheless, it is [incredibly configurable](/configuration). To get started you only need to understand four **Core Concepts**:
+Since v4.0.0, webpack does not require a configuration file. Nevertheless, it is [incredibly configurable](/configuration). To get started you only need to understand four **Core Concepts**:
```
Comment (EugeneHlushko) [inline src/content/concepts/index.md]: or `Webpack v4 and onwards does not...`

Comment (nveenjain) [reply src/content/concepts/index.md]: Actually @EugeneHlushko , webpack should be written with lowercase `w` https://webpack.js.org/branding/#the-name

Comment (EugeneHlushko) [reply src/content/concepts/index.md]: I dont mind as far as it gets us to `Better readability`

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**